actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -499,46 +499,44 @@ def apply_form_for_options!(object, options) # :nodoc:
       #   <% end %>
       #   # =>
       #   <form method="post">
-      #     <input type="text" name="title">
+      #     <input type="text" name="title" />
       #   </form>
       #
       #   # Adding a scope prefixes the input field names:
-      #   <%= form_with scope: :post, url: posts_path do |form| %>
+      #   <%= form_with scope: :article, url: articles_path do |form| %>
       #     <%= form.text_field :title %>
       #   <% end %>
       #   # =>
-      #   <form action="/posts" method="post">
-      #     <input type="text" name="post[title]">
+      #   <form action="/articles" method="post">
+      #     <input type="text" name="article[title]" />
       #   </form>
       #
       #   # Using a model infers both the URL and scope:
-      #   <%= form_with model: Post.new do |form| %>
+      #   <%= form_with model: Article.new do |form| %>
       #     <%= form.text_field :title %>
       #   <% end %>
       #   # =>
-      #   <form action="/posts" method="post">
-      #     <input type="text" name="post[title]">
+      #   <form action="/articles" method="post">
+      #     <input type="text" name="article[title]" />
       #   </form>
-      #
       #   # An existing model makes an update form and fills out field values:
-      #   <%= form_with model: Post.first do |form| %>
+      #   <%= form_with model: Article.first do |form| %>
       #     <%= form.text_field :title %>
       #   <% end %>
       #   # =>
-      #   <form action="/posts/1" method="post">
-      #     <input type="hidden" name="_method" value="patch">
-      #     <input type="text" name="post[title]" value="<the title of the post>">
+      #   <form action="/articles/1" method="post">
+      #     <input type="hidden" name="_method" value="patch" />
+      #     <input type="text" value="<the title of the article>" name="article[title]" />
```

Comment:
Just noting that while the order may be rendered this way, it's sort of irrelevant (and it used to be sorted iirc)

Reply:
Hi Carlos

I'll put it back in the original order then?